src/scales/quantitativeScale.ts

```diff
@@ -123,7 +123,7 @@ module Plottable {
     }
 
     private _includeValues(domain: D[]) {
-      this._includedValues.values().forEach((value) => {
+      this._includedValues.forEach((value: D) => {
```

Comment:
Can't be implicitly defined because the function parameters are not defined. Let me check if I can change that